### ISSUE TITLE
Version 2.12.1.527 is out of beta 

### ISSUE THREAD

**Issue by PkW4v3** [ddvk/remarkable-hacks#412]
Version 2.12.1.527 is now available on the RM1 💃🔝🔝

**Comment by ekatsili** [comment]
Just the RM1 for now?

**Comment by Eeems** [comment]
It was released for both the rM1 and rM2.

**Comment by ekatsili** [comment]
I get the following message when I attempt to install:

The version the device is running is not supported, yet. 20220202133838

I have checked my device and am definitely running 2.12.1.527

**Comment by Eeems** [comment]
> I get the following message when I attempt to install:
> 
> The version the device is running is not supported, yet. 20220202133838
> 
> I have checked my device and am definitely running 2.12.1.527

The OS versions are out, hacks has not been updated yet to support them. This issue was opened to alert @ddvk that a new OS is out.

**Comment by nhadfield** [comment]
Is the hack compatible with the new OS?

**Comment by CrazyEasy** [comment]
> Is the hack compatible with the new OS?

Not yet, we are waiting for @ddvk to release an updated version of the hack!

**Comment by nhadfield** [comment]
Got it - thanks!  I need to remember to wait for the hack before updating!

**Comment by CrazyEasy** [comment]
> Got it - thanks! I need to remember to wait for the hack before updating!

Yes, you should disable automatic updates, until @ddvk released an updated version 😉

**Comment by nhadfield** [comment]
Out of curiosity...will all of my bookmarks be lost now?

**Comment by Eeems** [comment]
> Out of curiosity...will all of my bookmarks be lost now?

Historically, bookmarks are properly imported between updates.

**Comment by nhadfield** [comment]
@ddvk agreed - thanks you so much for your work on this!!

**Comment by RobFreundlich** [comment]
> I'll try to find some time in the next days...

Thank you! I forgot to turn off updates, and now I'm back using plain vanilla RM2. It's still a great product, but wow, do I miss having your hacks! I don't say that to nag you, but rather to say how much I appreciate what you've created. So I'll just end as I began: Thank you!

**Comment by DizaBlah** [comment]
I too should have waited! But it's nice to get back to vanilla to remind us
of the awesomeness of the DDVK hacks!

+* mendizabal*
= *digital stuntman + solutionist*

*    646.450.7222*
    *mndzbl.com* <http://mndzbl.com/>
